tools/devctr/pyproject.toml

```diff
@@ -5,22 +5,28 @@ description = "Firecracker python dependencies"
 authors = ["Your Name <you@example.com>"]
 
 [tool.poetry.dependencies]
+aws-embedded-metrics = "*"
 black = "*"
 boto3 = "*"
 botocore = "*"
 dataclasses = "*"
 gitlint = "0.16.0"
+less = "*"
```

Comment:
This should go in the Dockerfile / apt part. Funny there is also a less in PyPi.

Reply:
oh... yeah... that's right... it's the tool, not the dependency what we need; fixing that

Reply:
I fixed that (and also updated the docker image).